tests/config/singleprovider.e2e.template.config.yaml

```diff
@@ -17,4 +17,5 @@ ols_config:
   default_provider: $PROVIDER
   default_model: $MODEL
 dev_config:
-  llm_temperature_override: 0
+  llm_params:
+    temperature: 0.05
```

Comment:
please put this back to 0, my guess is it'll just work since now you're merging the entire params dict instead of checking if the temperature field is set.  But if not, please add whatever other changes are needed so that a temperature of zero is respected + used.